test-data/unit/check-expressions.test

```diff
@@ -182,6 +182,22 @@ main:3: error: Unsupported operand types for * ("A" and "C")
 main:4: error: Incompatible types in assignment (expression has type "C", variable has type "A")
 main:5: error: Unsupported left operand type for * ("B")
 
+[case testMatMul]
```

Comment:
I have added a test to verify that the fast parser recognizes the operator. As far as I understand there's nothing more to test there, not more than there is for any other operator.